tsrc/cli/push_gitlab.py

```diff
@@ -94,6 +103,12 @@ def _get_group(self, group_name: str) -> Optional[Group]:
             else:
                 raise
 
+    def check_gitlab_feature(self, name: str) -> None:
+        features = self.gitlab_api.features.list()
```

Comment:
note: don't worry, the request is cached under the hood by the `python-gitlab` library. This is fine